ecs-configuration/logstash/pipelines/suricata/logstash-601-filter-suricata-smb.conf

```diff
@@ -9,10 +8,13 @@ if [@metadata][stage] == "suricata_json" {
           "[smb][created]" => "[file][created]"
           "[smb][modified]" => "[file][modified]"
           "[smb][filename]" => "[file][name]"
-        }
-        merge => { "[related][id]" => "[log][id][fuid]" }
-      }
+          "[smb][share]" => "[file][path]"
+          "[smb][share_type]" => "[file][type]"
```

Comment:
These are smb share specific, so maybe let's leave these for now. the `file.path` would be relative to the share, so it's like two different paths to combine to really find the file.

Reply:
Not sure what you are asking to be done with this one.